src/ui/flutter_app/lib/mill/recorder.dart

```diff
@@ -157,11 +149,10 @@ class GameRecorder {
     return false;
   }
 
-  String playOkToWmdMoveList(String playOk) {
-    return "";
-  }
+  String? import(String moveList) {
+    clear();
```

Comment:
TODO: I don't know where the function `clear` came from.

Reply:
see above (separate branch)